contribute/documentation.md

```diff
@@ -7,7 +7,6 @@ title: Documentation Contributions
 There are several ways you can help out with the improvement of Scala documentation. These include:
 
 * API Documentation in Scaladoc
-* Code examples and tutorials in activator templates.
```

Comment:
For the getting started page, you can point to http://www.scala-sbt.org/0.13/docs/sbt-new-and-Templates.html